packages/next/src/build/collect-build-traces.ts

```diff
@@ -650,8 +644,7 @@ export async function collectBuildTraces({
         }
 
         // edge routes have no trace files
-        const pageInfo = infos.get(route)
-        if (pageInfo?.runtime === 'edge') {
+        if (edgeRuntimeRoutes.includes(route)) {
```

Comment:
I was wrong it should probably be a plain object like

type RoutesUsingEdgeRuntime = {
  [route: string]: 0
}

it's a little silly but its basically a serializable set. would make the lookup here faster for large projects

Reply:
Though this makes reading the key more annoying since you'd have to do `edgeRuntimeRoutes.hasOwnProperty(route)`

Reply:
btw, this is exactly what we did in React when we introduced serializable state with prerender/resume. Feels funky to write but best perf while maintaining serializability

Reply:
Updated to follow your suggestion 👍